### DISCUSSION TITLE
Bug?: Goal tracking not recording goals

### DISCUSSION THREAD

**Discussion by wpdev8195** [plausible/analytics#2293] [Anything Else]
I am making pageview type goals. I tried making the goals using the format indicated on this page, https://plausible.io/docs/pageview-goals, using * or **.

Problem: On two of my pages, when I include the * at the end of the URL, the goal does not register.

Please see the attached screenshots for 1) a list of the goals being tested, and 2) a list of the pages visited and the goals that registered as a result. Note I visited some pages multiple times while testing, so please ignore the # of visits each page received and check only whether the page received any visits or not, and whether it received any goal hits or not.

![Screenshot 2022-10-03 101415](https://user-images.githubusercontent.com/108739698/193521039-5ef5fda1-41ba-496a-9db3-1c2d1a8565c0.png)
![Web capture_3-10-2022_10219_plausible io](https://user-images.githubusercontent.com/108739698/193521554-150b3797-3dbb-4e20-8df8-0c7a41f9e25d.jpeg)

**Comment by metmarkosaric** [comment]
hi @wpdev8195! the goals will only show in the dashboard when there's the first conversion. please share the url of your site as it's difficult to check without it. if you prefer not to share it publicly, do contact our customer support https://plausible.io/contact

**Comment by wpdev8195** [comment]
@metmarkosaric  The site is [url removed]. There are still goals that have not fired even though there have been conversions. The existence of the conversions can be verified by checking that the URL has been visited. The site is a WP site using the Plausible plugin.

**Reply by metmarkosaric** [reply]
thanks @wpdev8195! i just checked and reapplied those same goals and now they show up. not sure why they didn't show up in the first place but you're all good to go now

**Reply by wpdev8195** [reply]
That's strange! I removed them and re-added them several times and was not able to get them to show up. I do see they're showing up now. Thanks! 

Is it possible there's some bug where the user-added goals behave differently from the admin-added goals?

**Reply by metmarkosaric** [reply]
weird! i haven't encountered (his before so not sure. i've asked our developers to see if they had any thoughts on this but i'm glad to hear it's all working fine now
